hls4ml/backends/fpga/passes/clone.py

```diff
@@ -87,7 +87,7 @@ def transform(self, model, node):
                 )
                 for i in range(len(output_map[output])):
                     key = output + '_cpy' + str(i + 1)
-                    clone_layer.attributes[key].type = node.attributes['result_t']
+                    clone_layer.attributes[key].type = node.get_output_variable().type
```

Comment:
Isn't this identical?

Reply:
Here the output of the tests if you run the code with the original line:
```
    def match(self, node):
        if isinstance(node, (Pooling1D, Pooling2D, GlobalPooling1D, GlobalPooling2D)) and node.get_attr('pool_op') == 'Max':
>           return isinstance(node.get_input_variable().type.precision, XnorPrecisionType) and not isinstance(
                node.get_output_variable().type.precision, XnorPrecisionType
            )
E           AttributeError: 'FixedPrecisionType' object has no attribute 'precision'

hls4ml/backends/fpga/passes/xnor_pooling.py:14: AttributeError
```
Solved if you update it as proposed.